test/headers.js

```diff
@@ -212,7 +212,8 @@ test('buffer as `options.body` sets `content-length` header', async t => {
 	t.is(Number(headers['content-length']), buffer.length);
 });
 
-test('removes undefined value headers', async t => {
+// Todo: Enable this when node-fetch allows for removal of default headers
```

Comment:
```suggestion
// TODO: Enable this when node-fetch allows for removal of default headers
```

and link to the relevant issue or commit.